MediaBrowser.Plugins.Anime/MediaBrowser.Plugins.Anime.csproj

```diff
@@ -1,23 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;</TargetFrameworks>
     <AssemblyVersion>1.3.2.0</AssemblyVersion>
     <FileVersion>1.3.2.0</FileVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <None Remove="Configuration\configPage.html" />
-    <None Remove="thumb.png" />
   </ItemGroup>
 
   <ItemGroup>
     <EmbeddedResource Include="Configuration\configPage.html" />
-    <EmbeddedResource Include="thumb.png" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="mediabrowser.server.core" Version="3.6.0.40-beta" />
+    <PackageReference Include="jellyfin" Version="10.0.0" />
```

Comment:
This should be "Jellyfin.Controller"

Reply:
Cool, will do this when I get hone again :)

Reply:
I think the netstandard problem I had will go away with this :)

Reply:
Changed it :)